components/uuf-core/src/test/java/org/wso2/carbon/uuf/core/AppTest.java

```diff
@@ -104,8 +108,12 @@ public void testRenderPage() {
                                       emptySet(), null);
         Component rootComponent = new Component("root", null, Component.ROOT_COMPONENT_CONTEXT_PATH, emptySortedSet(),
                                                 emptySet(), emptySet(), singleton(cmp), null);
+        Configuration configuration = createConfiguration();
```

Comment:
Moved the creation of the `PluginProvider` to a separate method. Please check [createPluginProvider(Configuration configuration)](https://github.com/wso2/carbon-uuf/pull/241/files#diff-daad0991965a1cf287df6f89cfb139ebR101)